### DISCUSSION TITLE
How to access current item data fields in Select?

### DISCUSSION THREAD

**Discussion by albertbronsky** [mantinedev/mantine#703] [Q&A] [answered]
When using Mantine's `<Select>` component, is there a way to access the currently selected item?

I am trying to render the `icon` prop in `<Select>` but for that I need to fetch the `image` field of current item from `data` array.

**Comment by rtivital** [comment, marked as answer]
You control both data and value, and thus you can find the selected item in your data array:
```tsx
data.find(item => item.value === value)
```
